webpack/components/extensions/HostDetails/Tabs/RemoteExecutionActions.js

```diff
@@ -0,0 +1,35 @@
+import { API_OPERATIONS, post } from 'foremanReact/redux/API';
+import REX_JOB_INVOCATIONS_KEY from './RemoteExecutionConstants';
+import { foremanApi } from '../../../../services/api';
+import { getResponseErrorMsgs } from '../../../../utils/helpers';
+import { renderTaskStartedToast } from '../../../../scenes/Tasks/helpers';
+
+const errorToast = (error) => {
+  const message = getResponseErrorMsgs(error.response);
+  return message;
+};
+
+const jobInvocationParams = ({ hostname, packageName }) => ({
+  job_invocation: {
+    feature: 'katello_package_install',
+    targeting_type: 'Static Query',
```

Comment:
Does targeting_type need to be required? Nevertheless I'm not sure 'Static Query' is a valid option - I thought it was something like 'static' ? Could be wrong but want to confirm

Reply:
I removed it and it still works fine, so guess not! :+1: